app/api/gradeSimulation/create/func.js

```diff
@@ -0,0 +1,32 @@
+const app = require('@/app')
+const errors = require('@/errors')
+
+module.exports = async function (context) {
+  const GradeSimulation = app.model.gradeSimulations;
+  const HistoryGraduation = app.model.historiesGraduations;
+  const { historyGraduationId, subject, grade, season } = context.body
+
+  app.helpers.validate.throwMissingParameter(['historyGraduationId', 'subject', 'grade', 'season'], context.body)
+
+  const historyGraduation = await HistoryGraduation.findOne({ _id: String(historyGraduationId) })
+
+  if (!historyGraduation)
+    throw new errors.BadRequest(
+      `Histórico não encontrado: ${historyGraduationId}`
+    );
```

Comment:
Codacy has a fix for the issue: [Extra semicolon.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555384)
```suggestion
    )
```